src/filesystem/impls/filer/lib/is-animated/.npmignore

```diff
@@ -0,0 +1,9 @@
+# Coverage directory used by tools like istanbul
```

Comment:
@gideonthomas how do you want to handle importing this code?  I'm tempted to just take the code we need, and drop the rest of these files.

Reply:
yeah, I agree (I added a comment above), let's just take the code we need, add it to thirdparty and load it from there.